lib/project_types/script/layers/infrastructure/languages/assemblyscript_task_runner.rb

```diff
@@ -47,6 +47,17 @@ def metadata
             Domain::Metadata.create_from_json(@ctx, raw_contents)
           end
 
+          def library_version
+            # We should use something like:
+            # ep_config.type.language.package
```

Comment:
I don't think we should try to parse this from the user's `npm` state or environment, since we can't rely on it. 

I think we should use the same data source that we use to populate `extension_point_name`. We can then use this to read from `extension_point.yml`. That way, the way we interpret the extension point is consistent.

Reply:
>I think we should use the same data source that we use to populate extension_point_name. We can then use this to read from extension_point.yml.

Sorry, do you mean to just read it from `extension_point.yml`, because that's where we read the `extension_point_name`?